services/users/src/main/java/org/opfab/users/services/UserServiceImp.java

```diff
@@ -48,6 +48,18 @@ public class UserServiceImp implements UserService {
     @Autowired
     private RabbitTemplate rabbitTemplate;
 
+    /** Retrieve user information from repository for the user **/
```

Comment:
remove this comment and precise the method name if needed 
Do the same for existing method comments

Reply:
I think the names of the functions are already explicit so I removed the comments.